snapcraft/internal/repo/_deb.py

```diff
@@ -706,7 +696,10 @@ def _get_local_sources_list():
     return sources
 
 
-def _format_sources_list(sources_list: str, *, deb_arch: str, release: str = "xenial"):
+def _format_sources_list(sources_list: str):
+    release = os_release.OsRelease().version_codename()
+    deb_arch = snapcraft.ProjectOptions().deb_arch
```

Comment:
I wonder if this can go away completely.

Reply:
done, i ditched $suffix and $prefix.  I was originally going to wait on that, but it's here now. :D